helix-view/src/editor.rs

```diff
@@ -1715,3 +1715,106 @@ fn try_restore_indent(doc: &mut Document, view: &mut View) {
         doc.apply(&transaction, view.id);
     }
 }
+
+static EXPAND_ARGS_REGEXP: once_cell::sync::Lazy<helix_core::regex::Regex> =
+    once_cell::sync::Lazy::new(|| {
+        helix_core::regex::Regex::new(r"%(\w+)\{([^{}]*(\{[^{}]*\}[^{}]*)*)\}").unwrap()
+    });
+
+pub fn expand_args<'a>(editor: &Editor, args: &'a str) -> anyhow::Result<Cow<'a, str>> {
+    let (view, doc) = current_ref!(editor);
+    let shell = &editor.config().shell;
+
+    replace_all(
+        once_cell::sync::Lazy::force(&EXPAND_ARGS_REGEXP),
```

Comment:
```suggestion
        &EXPAND_ARGS_REGEXP,
```

Reply:
Also, a minor nit that this constant should probably be called `REGEX` rather than `REGEXP` to be consistent with regex in the regex crate and other usages in Helix